src/components/Header/index.js

```diff
@@ -1,97 +1,99 @@
-import React, { useState } from 'react'
+import React, { Fragment, useState, useEffect } from 'react'
 
 import styled from 'styled-components'
 import Flex from 'styled-flex-component'
 import { Padding } from 'styled-components-spacing'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
 
+import navLinks from './navLinks'
 import { Row, Col, Grid } from '../grid'
 import Hamburger from './Hamburger'
 import Overlay from './Overlay'
 import TopNav from './TopNav'
 import SideNav from './SideNav'
 
-const FixedHeightFlex = styled(Flex)`
-  height: 84px;
+const StyledPadding = styled(Padding)`
+  display: flex;
+  height: ${remcalc(120)};
```

Comment:
![Screen Shot 2019-05-02 at 10 13 55](https://user-images.githubusercontent.com/10553003/57066032-0b913900-6cc3-11e9-83cb-92ce73cfee99.png)
☝️  Without it the logos + link container don't align properly on mobile to smalltablet views.
By the way, the 120 is 84 + 36 (padding). It's a bit tricky because our header always has the same height (84) but the page/main/layout has a 36 padding bottom (and I actually had to fill the page/main/layout with an extra 36px at the top so if there is a color like blueBg color like it happens in specialities it reaches  the bottom of the TopNav). Hopefully I explained this properly.

Reply:
Firstly, I don't think we need 36px padding on this component any more. So the height will only ever be 84px, not 120px. Right?

And I get that you need a fixed height, in order for the centering to work :+1:  But think about it, why should you need a fixed height on a _`Padding`_ component?

Presumably the centering issue is why [you've added a fixed height to the overall Header (i.e. the StyledGrid)](https://github.com/yldio/yld.io/pull/189/files/98ec6f77a36ac39073036209d45f8eda1b41497a#diff-ff918d7f5a60607376b021ad2f84bc53R24)?

I believe that the place you actually need a fixed height is the StyledTopNavContainer?